lib_xud/src/core/XUD_IoLoop.S

```diff
@@ -200,12 +199,15 @@ ConfigRxA:                                      // Configure a event on RXA goin
     setv        res[RXA], r11
     eeu         res[RXA]
 
-ConfigValidToken:
 #ifdef __XS2A__
-    ldw         r10, sp[STACK_VTOK_PORT]        // ValidToken is used for timing timeout period when expecting an ack after tx
-#else
-    ldw         r10, dp[rx_rdy]                 // TODO use from stack
+ClearValidToken:                               // Clear any VTok port condition
```

Comment:
Bit confusing naming - maybe follow the previous naming convention - ConfigValidToken?

Essentially this part of the code is configuring port events/conditions.